src/__tests__/resetPassword.test.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-console */
```

Comment:
could you remove this line which disable the eslint rule

Reply:
I have removed it